dev tools (#46)
eth-cscs/cscs-docs: docs/software/devtools/perf/index.md

[](){#ref-software-devtools-perf}

> # Performance Analysis tools

Reviewer: ```suggestion
# Performance analysis tools
```

Author: see https://github.com/eth-cscs/cscs-docs/pull/46#discussion_r1998545752

Learning to analyze the performance of an applications effectively is crucial
to build a deeper understanding of how your code interacts with the underlying
hardware. In this section we introduce the various performance analysis
solutions available at CSCS. If you have issues or questions about performance

> analysis tools, please do not hesitate to contact us on
> [support.cscs.ch](https://support.cscs.ch).

Reviewer: See @msimberg suggestion above:
```suggestion
analysis tools, please do not hesitate to [contact us][ref-get-in-touch].
```
